plugins/weida-raw/fu-weida-raw-common.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 Randy Lai <randy.lai@weidahitech.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+const gchar *
+fu_weida_raw_strerror(guint8 code);
+
+gboolean
+fu_weida_raw_block_is_empty(const guint8 *data, gsize datasz);
+
+guint16
+fu_weida_raw_misr_16b(guint16 current_value, guint16 new_value);
+
+guint16
+fu_weida_raw_misr_for_bytes(guint16 current_value, guint8 *bytes, guint start, guint size);
+
+guint16
+fu_weida_raw_misr_for_halfwords(guint16 current_value, guint8 *buf, guint start, guint hword_count);
```

Comment:
Can you use fu_misr16() instead of this please

Reply:
Hi, In my branch(my_new_change) we don't have fu_misr16() function
I think I need to sync and merge the project or cherry-pick from main branch,
But I have no relevant experience during the pull request.

Could you tell me how best to do this?

Reply:
Sure! You can rebase -- e.g. something like:

    git remote add upstream
    git fetch upstream
    git rebase upstream/main

Reply:
thanks, soloved